pkg/cluster/nodes/node.go

```diff
@@ -76,17 +76,20 @@ func (n *Node) String() string {
 // SignalStart sends SIGUSR1 to the node, which signals our entrypoint to boot
 // see images/node/entrypoint
 func (n *Node) SignalStart() error {
-	return docker.Kill("SIGUSR1", n.nameOrID)
+	return container.Kill("SIGUSR1", n.nameOrID)
 }
 
 // CopyTo copies the source file on the host to dest on the node
 func (n *Node) CopyTo(source, dest string) error {
-	return docker.CopyTo(source, n.nameOrID, dest)
+	return container.CopyTo(source, n.nameOrID, dest)
 }
 
-// WaitForDocker waits for Docker to be ready on the node
-// it returns true on success, and false on a timeout
+// WaitForDocker waits for Docker Daemon to be ready on the node
+// it returns true on success or if you are not using Docker, and false on a timeout
```

Comment:
This function also should not change, this is waiting for the docker service inside the node container to be ready. 

This is the change that is causing the CI to fail.

Reply:
Ok, I guess if you were using podman on the host and docker within the container this would be an issue.  But this clearly shows for this use case why podman inside of the container would be better, no daemon to start and wait for.

Reply:
Sure, but podman does not implement a CRI for Kubernetes as far as I know. Docker in the container is purely for that use case.

We will be exploring cri-o as an option soon. @runcom started looking into this.